public.package.json

```diff
@@ -10,9 +10,27 @@
   },
   "type": "module",
   "exports": {
-    ".": "./index.js",
+    ".": {
+      "import": {
+        "types": "./index.d.ts",
+        "default": "./index.js"
+      },
+      "require": {
+        "types": "./index.d.ts",
+        "default": "./index.js"
+      }
+    },
```

Comment:
Aren't these two equivalent? Based on https://www.typescriptlang.org/docs/handbook/esm-node.html#packagejson-exports-imports-and-self-referencing I expect TS to find type files that are colocated. 

Maybe the `types` property on line 39 is interfering with that?

Reply:
I don't think that line 39 is an issue since it keeps the backwards compatibility for older versions of TypeScript. But you're right, `"types"` is redundant here.

Reply:
In fact, we don't need to make any changes to this file :)